source/schemes.ttl

```diff
@@ -246,182 +250,299 @@
     skos:broadMatch <http://id.loc.gov/vocabulary/subjectSchemes/mesh> ;
     foaf:homepage <https://mesh.kib.ki.se/> .
 
-
 # Aliases for external SubjectSchemes
 
 #@base <https://id.kb.se/ext/term/> . # ?
+# add @sv on titles in other languages or keep an untagged version?
 
 <agrovoc> a :TopicScheme;
     skos:notation "agrovoc";
-    dc:title "AgroVoc"@en;
-    skos:exactMatch <http://id.loc.gov/vocabulary/subjectSchemes/agrovoc>;
+    dc:title "AGROVOC Multilingual Thesaurus"@en;
+    dc:title "AGROVOC Multilingual Thesaurus";
+    foaf:page <http://aims.fao.org/vest-registry/vocabularies/agrovoc>;
+    skos:exactMatch <http://id.loc.gov/vocabulary/subjectSchemes/agrovoc>, <http://aims.fao.org/aos/agrovoc>;
     skos:scopeNote "Kan användas även i genre/form."@sv .
 
+<allars> a skos:TopicScheme;
+    skos:notation "allars";
+    dc:title "Allärs: allmän tesaurus pä svenska"@sv;
+    rdfs:comment "Nationalbiblioteket i Finland"@sv;
+    skos:exactMatch <http://id.loc.gov/vocabulary/subjectSchemes/allars>;
+    skos:broadmatch <http://www.yso.fi/onto/allars/> .
+
 <bella> a skos:TopicScheme;
     skos:notation "bella";
     dc:title "Bella: specialtesaurus för skönlitteratur"@sv;
     skos:exactMatch <http://id.loc.gov/vocabulary/subjectSchemes/bella> .
 
+<bokbas> a skos:TopicScheme;
+    skos:notation "bokbas";
+    dc:title "Bokbasen"@no;
+    dc:title "Bokbasen";
+    skos:exactMatch <http://id.loc.gov/vocabulary/subjectSchemes/bokbas> .
+
 <bnb> a skos:ConceptScheme;
     skos:notation "bnb";
-    dc:title "British National Bibliography"@en .
+    dc:title "British National Bibliography"@en;
+    dc:title "British National Bibliography" .
+
+<czenas> a skos:TopicScheme;
+    skos:notation "czenas";
+    dc:title "CZENAS thesaurus: a list of subject terms used in the National Library of the Czech Republic"@en;
+    dc:title "Soubor vĕcných autorit Národní knihovny ČR"@cs;
+    dc:title "CZENAS thesaurus";
+    rdfs:comment "Tjeckien, Prag: Národní knihovna Čseké republiky"@sv;
+    skos:exactMatch <http://id.loc.gov/vocabulary/subjectSchemes/czenas> .
 
 <fast> a skos:ConceptScheme;
     skos:notation "fast";
     dc:alternative "FAST";
     dc:title "Faceted application of subject terminology"@en;
+    dc:title "Faceted application of subject terminology";
+    rdfs:comment "Dublin, Ohio: OCLC"@sv;
     skos:scopeNote "Kan användas även i genre/form"@sv;
     skos:exactMatch <http://id.loc.gov/vocabulary/subjectSchemes/fast>;
-    foaf:homepage <http://fast.oclc.org/> .
+    foaf:page <http://fast.oclc.org/> .
+
+<gtt> a skos:TopicScheme;
+    skos:notation "gtt";
+    dc:title "GOO-trefwoorden thesaurus"@en;
+    dc:title "GOO-trefwoorden thesaurus";
+    rdfs:comment "Nederländerna, Haag: Koninklijke Bibliotheek"@sv;
+    skos:exactMatch <http://id.loc.gov/vocabulary/subjectSchemes/gtt> .
+
+<humord> a skos:TopicScheme;
+    skos:notation "humord";
+    dc:title "HUMORD";
```

Comment:
Why not: `dc:title "HUMORD"@no, dc:title "HUMORD";` ?

Reply:
Was not sure because of the CAPITALS, I think we need to follow up on some of these soon anyway, so I think it is fine for now.